### ISSUE TITLE
[Question] How to use Jasmine with Jest-Playwright?

### ISSUE THREAD

**Issue by jevgenijusmarinuskinas** [microsoft/playwright#4873]
Hello,

I was using Jasmine to add reporters and get name of a test with Puppeteer. After Playwright migration, it is no longer working. 

My jest.setup.js looks like this:

```
jasmine.getEnv().addReporter({
    specStarted: result => jasmine.currentTest = result,
    specDone: result => jasmine.currentTest = result,
    suiteStarted: result => jasmine.currentSuite = result,
    suiteDone: result => jasmine.currentSuite = result,
});
````

Whenever I try to execute some tests, I am getting an error:

````
   ReferenceError: jasmine is not defined

    > 1 | jasmine.getEnv().addReporter({
        | ^
      2 |     specStarted: result => jasmine.currentTest = result,
      3 |     specDone: result => jasmine.currentTest = result,
      4 |     suiteStarted: result => jasmine.currentSuite = result,

      at Object.<anonymous> (test-environment/jest.setup.js:1:1)
````

Is there any possible solution?

**Comment by aslushnikov** [comment]
Hey @jevgenijusmarinuskinas, both Puppeteer and Playwright are not related to jest/jasmine, so migrating one to another couldn't break anything there.

Do you use `jest-playwrigh` or something similar?

**Comment by jevgenijusmarinuskinas** [comment]
Hello. 
Yes, actually, before it was jest-puppeteer, jest-environment-puppeteer. I also understand, that it should not be directly related, but after migratiin to jest-playwright, this became broken.

Before, calling jasmine was globally available across the modules, without any additional imports, just as it is seen in the content of file in my first message.

After changing puppeteer dependencies with playwright, it became undefined.

Basically, what it was used for in test suites, is to get the name if the current test in beforeEach, and for similar reasons in other places. So I am looking for a way to make it work again with Playwright or for something to replace it with.

**Comment by aslushnikov** [comment]
@joeleinbinder: any chance you know what's going on in jest-playwright and jasmine?

**Comment by aslushnikov** [comment]
(also cc'ing the jest-playwright expert): @mxschmitt, any ideas?

**Comment by pavelfeldman** [comment]
Please talk to jest-playwright folks in their repo!
